doc/fvwm/options.xml

```diff
@@ -93,8 +92,9 @@ running.</para>
 <replaceable>config-file</replaceable>
 instead of
 <filename>~/.fvwm/config</filename>
-as its initialization file.  This is equivalent to
--c '<fvwmref cmd="Read"/>
+as its initialization file. <envar>$FVWM_USERDIR</envar> can also be used to 
+change location of default user directory <filename>~/.fvwm</filename>. Another
+option is to use -c '<fvwmref cmd="Read"/>
```

Comment:
I'm not in favour of advocating `-c` -- it's one of those options which will go...